xapian-applications/omega/query.cc

```diff
@@ -1307,6 +1309,15 @@ eval(const string &fmt, const vector<string> &param)
 		value = str(collapsed);
 		break;
 	    }
+            case CMD_contains: {
+                size_t pos = args[1].find(args[0]);
+                if (pos != string::npos) {
+                    value = str(pos);
+                } else {
+                    value = "";
+                }
```

Comment:
@ayushgp As I mentioned in feedback on the original PR, `value` starts empty, so this `else` is unnecessary.  I'll merge and fix this in a follow-up commit.